demos/jans-tarp/src/options/oidcClients.tsx

```diff
@@ -123,17 +133,22 @@ function Row(props: { row: ReturnType<typeof createData> }) {
     );
 }
 
-export default function OIDCClients(data) {
+export default function OIDCClients({ data, notifyOnDataChange }) {
     const [modelOpen, setModelOpen] = React.useState(false);
     const [drawerOpen, setDrawerOpen] = React.useState(false);
     const handleDialog = (isOpen) => {
         setModelOpen(isOpen);
+        notifyOnDataChange();
     };
 
     const handleDrawer = (isOpen) => {
         setDrawerOpen(isOpen);
     };
 
+    const handleNotifyOnDataChange = () => {
```

Comment:
This function is unnecessary, you can directly use notifyOnDataChange function.

Reply:
done